frontend/express/public/javascripts/countly/vue/components/sidebar.js

```diff
@@ -7,15 +7,23 @@
             template: CV.T('/javascripts/countly/vue/templates/sidebar/AppSelector.html'),
             data: function() {
                 return {
-                    activeApp: {
-                        name: "",
-                        value: ""
-                    },
-                    value: "",
                     centerDialogVisible: true,
                     app_selector__dialog_box: "cly-vue-sidebar__app-selector__dialog_box"
                 };
             },
+            computed: {
+                activeApp: {
+                    get: function() {
+                        var app = this.$store.getters["countlyCommon/getActiveApp"];
+                        var tempApp = Object.assign({data: 0}, app);
+                        tempApp["value"] = tempApp._id;
+                        return tempApp;
+                    },
+                    set: function(activeApp) {
+                        this.$store.dispatch("countlyCommon/updateActiveApp", activeApp.id);
```

Comment:
i dont think you need to do this. since you already are emitting the change event to parent.